### PR TITLE
* MDF [conf] update docs for NanoMQ basic conf

### PR DESCRIPTION
**Pull request by OdyWayne**

### CODE REVIEW

**Review by wanghaEMQ — APPROVED** [review]

File: docs/en_US/config-description/broker.md [hunk 1]
```diff
@@ -15,30 +15,37 @@ system {
...
 
 ### Configuration Items
 
-- `disk_cache_size`: Specifies the maximum number of messages that can be cached in the SQLite database; default: 102400; Optional Values:
-  - 0:  Ineffecitve
-  - 1 - infinity
-- `mounted_file_path`: Specifies the file path where the SQLite database file is mounted; default: `nanomq running path`
-- `flush_mem_threshold`: Specifies the threshold for flushing messages to the SQLite database. When the number of messages reaches this threshold, they are flushed to the SQLite database; default: 100, Value range: 1-infinity
-- `resend_interval`: (Currently not implemented) Specifies the interval, in milliseconds, for resending the messages after a failure is recovered. This is unrelated to the trigger for the resend operation; default: 5000. Note:  **Only work for the NanoMQ broker to resend cached messages to local client, not for bridging connections**.
+- `disk_cache_size`: Specifies the maximum number of messages that can be cached in the SQLite database.
```
Comment (JaylinYu) [inline docs/en_US/config-description/broker.md:42]: elaborate about retain persistence

**Review by JaylinYu — CHANGES_REQUESTED** [review]

File: docs/en_US/config-description/broker.md (hunk 1 above)
Comment (OdyWayne) [reply docs/en_US/config-description/broker.md:42]: ok

**Review by JaylinYu — APPROVED** [review]
🌝🌝

File: docs/zh_CN/tutorial/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# NanoMQ使用场景教程
+
+本节为常用场景提供了教程。
```
Comment (wanghaEMQ) [inline docs/zh_CN/tutorial/tutorial.md:3]: Is there anything more?

Comment (OdyWayne) [reply docs/zh_CN/tutorial/tutorial.md:3]: Not now. But we can add tutorial here, if there is more to come.

**Review by JaylinYu — COMMENTED** [review]
🌚🌚

File: docs/zh_CN/tutorial/tutorial.md (hunk 2 above)
Comment (wanghaEMQ) [reply docs/zh_CN/tutorial/tutorial.md:3]: Good

**Pull request merged**